helix-term/src/ui/mod.rs

```diff
@@ -265,6 +266,76 @@ pub fn file_picker(root: PathBuf, config: &helix_view::editor::Config) -> FilePi
     picker
 }
 
+pub fn file_browser(root: PathBuf) -> Result<FilePicker, std::io::Error> {
+    let root = helix_stdx::path::canonicalize(root);
```

Comment:
I'm fairly certain we don't need to canonicalize this path. Canonicalization is almost never the right thing to do and it usually causes weird effects with symlinks.